jerry-debugger/jerry-client-ws.html

```diff
@@ -671,6 +738,30 @@
         return;
       }
 
+      case JERRY_DEBUGGER_EVAL_RESULT:
+      case JERRY_DEBUGGER_EVAL_RESULT_END:
+      case JERRY_DEBUGGER_EVAL_ERROR:
+      case JERRY_DEBUGGER_EVAL_ERROR_END:
+      {
+        evalResult = concatUint8Arrays(evalResult, message);
+
+        if (message[0] == JERRY_DEBUGGER_EVAL_RESULT_END)
+        {
+          appendLog(cesu8ToString(evalResult));
+          evalResult = null;
+          return;
+        }
+
+        if (message[0] == JERRY_DEBUGGER_EVAL_ERROR_END)
+        {
+          appendLog("ERROR: " + cesu8ToString(evalResult));
```

Comment:
How about using "Uncaught: " here, and just going with `to_string` in case of error objects where you send the result? It would also signal that the thrown value is an object, and not just a thrown string.